src/components/Activity/activity-types.js

```diff
@@ -70,10 +70,6 @@ const ACTIVITY_TYPES = {
     icon: iconAnj,
     title: 'Settle appeal deposit',
   },
-  transaction: {
-    title: 'Transaction',
-    icon: iconAnj,
-  },
```

Comment:
I think we also need to remove the default in https://github.com/aragon/court-dashboard/pull/335/commits/119a2d7b9ae4cc7391888dc92cb63399a91bdf15#diff-cabff7987bb1e5e4650eb3786dac7c19R80